pyproject.toml

```diff
@@ -0,0 +1,56 @@
+[project]
+name = "python-stiebel-eltron"
+dynamic = ["version"]
+description = "Python API for interacting with the Stiebel Eltron ISG web gateway via Modbus for controlling integral ventilation units and heat pumps."
+readme = "README.md"
+authors = [
+    { name = "Manuel Stahl"},
+    { name = "'Martin Fuchs"}
+]
+license = { file = "LICENSE" }
+classifiers = [
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",   
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",    
+]
+requires-python = ">=3.10"
```

Comment:
Does not match the classifiers above

Reply:
I changed line 20 to `requires-python = ">=3.8"`

Reply:
I changed it again to r`equires-python = ">=3.10,<3.14"` as newer versions of pymodbus do not support python versions below 3.10. Therefor I also remove 3.8 and 3.9 in the table above.